noggin/security/ipa.py

```diff
@@ -35,6 +36,20 @@ class Client(IPAClient):
     TODO: send this upstream.
     """
 
+    def group_find(self, *args, **kwargs):
+        """
+        Find a group. A wrapped method to allow filtering of groups to be hidden from the user.
+
+        TODO: this needs to be retained when we migrate to the newer Client class.
+        """
+        hidden_groups = current_app.config.get('HIDDEN_GROUPS')
+        groups = [
+            g
+            for g in IPAClient.group_find(self, *args, **kwargs)['result']
```

Comment:
@abompard I think this is ready now if you get a chance. I still need to use the Class name directly for some reason. If its a big deal I can find out why I can't use super.